.github/workflows/sync-helm-readme.yml

```diff
@@ -0,0 +1,90 @@
+# Workflow to sync helm chart README.md when values.yml is updated
+name: Sync Helm Chart README.md
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
+    branches:
+      - 'main'
+    paths:
+      - 'charts/aws-pca-issuer/values.yaml'
+
+env:
+  GITHUB_USER_NAME: github-actions
+  GITHUB_USER_EMAIL: github-actions@github.com
+
+jobs:
+  sync-helm-chart-doc:
+    if: github.event.action != 'labeled' || github.event.label.name == 'sync helm readme'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.24'
+
+      - name: Run helm-docs via Makefile
+        id: run-helm-docs
+        run: |
+          mkdir -p $(pwd)/bin
+          export PATH=$(pwd)/bin:$PATH
+          echo "Regenerating charts/aws-pca-issuer/README.md since charts/aws-pca-issuer/values.yaml has updated"
+          make helm-docs
+          MAKE_EXIT_CODE=$?
+          if [ $MAKE_EXIT_CODE -eq 0 ]; then
+            echo "result=success" >> "$GITHUB_OUTPUT"
+            echo "Output set to: success"
+          else
+            echo "result=fail" >> "$GITHUB_OUTPUT"
+            echo "Output set to: fail"
+          fi
+          echo "Make exit code: $MAKE_EXIT_CODE"
+          exit $MAKE_EXIT_CODE
+
+      - name: Update pull request with new Helm README.md
+        if: ${{ steps.run-helm-docs.outputs.result == 'success' }}
```

Comment:
Is this the standard way to do this?

Reply:
While some of our other workflows use this method, I looked into this and used the standard `steps.stepname.outcome` instead here. This let me refactor the `make` step and eliminate some redundant lines. Updated the commit